ocrd_models/ocrd_models/ocrd_page.py

```diff
@@ -39,25 +70,58 @@
     parseString,
 
     AlternativeImageType,
+    BaselineType,
+    BorderType,
+    ChartRegionType,
+    ChemRegionType,
     CoordsType,
+    CustomRegionType,
     GlyphType,
+    GraphemeBaseType,
+    GraphemeGroupType,
+    GraphemeType,
+    GraphemesType,
+    GraphicRegionType,
+    GridPointsType,
+    GridType,
     ImageRegionType,
     LabelType,
     LabelsType,
+    LayerType,
+    LayersType,
+    LineDrawingRegionType,
+    MapRegionType,
     MathsRegionType,
+    MetadataItemType,
```

Comment:
This was the culprit (not importing what was exported via `__all__`). But I am fine with exporting everything, too.

Reply:
That way, I hope I will realize to update the exports/__all__ whenever generateDS changes.